src/c_api/c_api_symbolic.cc

```diff
@@ -658,6 +658,85 @@ int MXSymbolInferShapeEx(SymbolHandle sym,
   API_END();
 }
 
+int MXSymbolInferShapeExInt64(SymbolHandle sym,
+                              mx_uint num_args,
+                              const char** keys,
+                              const int64_t *arg_ind_ptr,
```

Comment:
There are mixed use of int64_t and mx_int64. I would suggest we consolidate them. If you prefer, I am okay to do it in a separate PR.

Reply:
Copy paste from the 32 bit function with very little differences.

Reply:
done